api/api/models/vulns.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+from api.models import Pentester
+
+MAX_NOTE_LENGTH = 8186
+
+
+class Notes(models.Model):
+    """
+        In order to give pentesters flexibility in their work, we should let them save simple notes about the
+        current infrastructure.
+    """
+
```

Comment:
please add `REQUIRED_FIELDS` so Swagger documentation stays up to date

Reply:
done